### PR TITLE
[Jetcaster] Upgrade to snapshot 7123292

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt [hunk 1]
```diff
@@ -139,7 +139,7 @@ fun EpisodeListItem(
                 top.linkTo(parent.top)
                 centerHorizontallyTo(parent)
 
-                width = Dimension.fillToConstraints
+                width = androidx.compose.foundation.layout.Dimension.fillToConstraints
```
Comment (florina-muntenescu) [inline Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt:142]: import

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt [hunk 2]
```diff
@@ -185,13 +185,13 @@ fun EpisodeListItem(
                 )
                 top.linkTo(parent.top, 16.dp)
 
-                width = Dimension.preferredWrapContent
+                width = androidx.compose.foundation.layout.Dimension.preferredWrapContent
```
Comment (yrezgui) [inline Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt:188]: Use import

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt [hunk 3]
```diff
@@ -206,7 +206,7 @@ fun EpisodeListItem(
                     )
                     top.linkTo(episodeTitle.bottom, 6.dp)
 
-                    width = Dimension.preferredWrapContent
+                    width = androidx.compose.foundation.layout.Dimension.preferredWrapContent
```
Comment (yrezgui) [inline Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt:209]: Use import

**Review by yrezgui — CHANGES_REQUESTED** [review]
I would ask @chrisbanes to review specifically animation changes

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt (hunk 2 above)
Comment (chrisbanes) [reply Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt:188]: Nope, using this means we can suppress only the function, not the entire file.

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt (hunk 1 above)
Comment (chrisbanes) [reply Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt:142]: Nope, using this means we can suppress only the function, not the entire file.

**Comment by chrisbanes** [comment]
> I would ask @chrisbanes to review specifically animation changes

I did those changes 😅

**Review by yrezgui — APPROVED** [review]

**Pull request merged**